app/models/agents/data_output_agent.rb

```diff
@@ -24,6 +24,8 @@ class DataOutputAgent < Agent
           * `template` - A JSON object representing a mapping between item output keys and incoming event values.  Use [Liquid](https://github.com/cantino/huginn/wiki/Formatting-Events-using-Liquid) to format the values.  Values of the `link`, `title`, `description` and `icon` keys will be put into the \\<channel\\> section of RSS output.  Value of the `self` key will be used as URL for this feed itself, which is useful when you serve it via reverse proxy.  The `item` key will be repeated for every Event.  The `pubDate` key for each item will have the creation time of the Event unless given.
           * `events_to_show` - The number of events to output in RSS or JSON. (default: `40`)
           * `ttl` - A value for the \\<ttl\\> element in RSS output. (default: `60`)
+          * `ns_media` - Add [yahoo media namespace](https://en.wikipedia.org/wiki/Media_RSS) in output xml (default: `true`)
```

Comment:
I'm setting `ns_media` to true when creating a new agent ([line 74](https://github.com/cantino/huginn/pull/1411/files#diff-3662477d85fa04658d57d3486c39ca5aR74)). 
Should I take that out and just leave both namespaces up to user?